README.md

```diff
@@ -28,6 +28,10 @@ git clone git://github.com/ets-berkeley-edu/suitec.git
 
 ## Dependencies
 
+### Canvas
+
+On the [Canvas LMS github README](https://github.com/instructure/canvas-lms) page, go to *Installation > Quick Start*. Follow those instructions to get Canvas LMS running on *http://localhost:3000*.
+
```

Comment:
@raydavis @paulkerschen  With new laptop, I rebuilt my local Canvas instance with instructions above. The instructions in CalCentral wiki are no longer reliable.  See [preview of this README change](https://github.com/johncrossman/suitec/tree/COL-1194_hidden-ok).